src/LandIce/evaluators/LandIce_HydrologyMeltingRate_Def.hpp

```diff
@@ -42,19 +44,19 @@ HydrologyMeltingRate (const Teuchos::ParameterList& p,
    * is in m/yr, the mesh is in km, and hydrology time unit is s.
```

Comment:
I'm no expert in ice thermodynamics, but I _assume_ they could be the same.

Separate topic: I also assume that L _may_ depend on pressure. I have no idea how much, or even if that is linked with whether the temperature used is the pressure-corrected one.

Anyhow, long story short, I think we can use the same parameter everywhere.

Reply:
OK, I'll cahnge the temperature solver to use the one you are using. I would have anyway to fix it, since it's not capitalized correctly

Reply:
Wait, I just changed it in the hydrology to use `Latent heat of fusion`. I think it's correct to mention 'of fusion', since there's also the evaporation latent heat. I guess we could do `Ice Latent Heat Of Fusion`, to stick with capitalization.

Reply:
Yeah, that would be the best. Should be easy to fine and replace that everywhere.

Reply:
Done.